ramls/oai-pmh.raml

```diff
@@ -0,0 +1,131 @@
+  list: !include rtypes/list.raml
+
+/oai:
+  /records:
+    displayName: Records
+    type:
+      list:
+        exampleCollection: !include examples/records.sample
+    get:
+      is: [ partionable, selective, metadataPrefix ]
+      responses:
+        404:
+          description: "Item with a given ID and selective harvesting params (if any) not found"
+          body:
+            application/xml:
+              schema: oai-pmh
+              example: !include examples/no_records_error.sample
+
+    /{id}:
+      uriParameters:
+        id:
+          description: The unique identifier of the item in the repository
+          type: string
+      get:
+        description: Retrieve record with given Id
+        is: [ metadataPrefix ]
+        responses:
+          200:
+            description: "Returns item with a given ID"
+            body:
+              application/xml:
+                example: !include examples/record.sample
+                schema: oai-pmh
+          404:
+            description: "Item with a given ID and selective harvesting params (if any) not found"
+            body:
+              application/xml:
+                schema: oai-pmh
+                example: !include examples/no_record_error.sample
+          500:
+            description: "Internal server error, e.g. due to misconfiguration"
+            body:
+              text/plain:
+                example: "Internal server error, contact administrator"
+
+  /identifiers:
+    displayName: Identifiers
+    type:
+      list:
+        exampleCollection: !include examples/identifiers.sample
+    get:
+      is: [ partionable, selective, metadataPrefix ]
+      responses:
+        404:
+          description: "Item with a given ID and selective harvesting params (if any) not found"
+          body:
+            application/xml:
+              schema: oai-pmh
+              example: !include examples/no_records_error.sample
+
+  /metadata_formats:
+    displayName: MetadataFormats
+    type:
+      list:
+        exampleCollection: !include examples/metadata_formats.sample
+    get:
+      queryParameters:
+        identifier:
+          description: 'unique identifier of the item in the repository from which the record must be disseminated'
+          type: string
+          required: false
+      responses:
+        404:
+          description: "Item with a given ID and selective harvesting params (if any) not found"
+          body:
+            application/xml:
+              schema: oai-pmh
+              example: !include examples/no_record_error.sample
+        422:
+          description: "The request was well-formed, but contains semantic errors"
+          body:
+            application/xml:
+              schema: oai-pmh
+              example: !include examples/no_metadata_format_error.sample
+
+  /sets:
+    displayName: Sets
+    type:
+      list:
+        exampleCollection: !include examples/sets.sample
+    get:
+      is: [ partionable ]
```

Comment:
Responses?  200, 500 should probably do it.

Reply:
/sets resource is of type 'list' which defines status codes 200 and 500 among other things